src/scss/base/_mixins.scss

```
@@ -0,0 +1,38 @@
+@mixin size($width, $height: $width) {
+  width: $width;
+  height: $height;
+}
+
+@mixin position($position, $top: null, $right: null, $bottom: null, $left: null) {
+  position: $position;
+  top: $top;
+  right: $right;
+  bottom: $bottom;
+  left: $left;
+}
+
+@mixin centerBox() {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin Mobile { @media (max-width: $mobile-width) {     
+  @content;   
+ } 
+}  
```

```suggestion
@mixin Mobile { 
  @media (max-width: $mobile-width) {     
    @content;   
   } 
}  
```
Check luôn ở các mixin còn lại
Ngoài ra có thể tham khảo thêm cách tạo mixin reponsive bằng ví dụ dưới đây
https://css-tricks.com/snippets/sass/mixin-manage-breakpoints/